tests/lib/astarte_device_sdk/unit/object/src/main.c

```diff
@@ -120,19 +120,19 @@ ZTEST(astarte_device_sdk_object, test_deserialize_astarte_object_from_aggregate)
 
     astarte_object_entry_t entry_double = entries[0];
     zassert_equal(strcmp(entry_double.path, test_data_double_path), 0);
-    astarte_individual_t individual_double = entry_double.individual;
+    astarte_data_t individual_double = entry_double.individual;
```

Comment:
Nit: What do you think to port all individual variable and field to `data`?
```suggestion
    astarte_data_t data_double = entry_double.data;
```